src/facilities.js

```diff
@@ -93,6 +93,48 @@ class Facilities {
       .then((facility) => this._formatFacilityFromServer(facility));
   }
 
+  /**
+   * Creates or updates a facility's info (NOTE: This refers to the facility_info model)
+   *
+   * API Endpoint: '/facilities/:facilityId/info?should_update=true'
+   * Method: POST
+   *
+   * @param {number} facilityId The id of the facility to update
+   * @param {Object} update An object containing the facility info for the facility
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.facilities.createOrUpdateInfo(25, {
+   *   square_feet: '10000'
+   * });
+   */
+  createOrUpdateInfo(facilityId, update) {
+    if (!facilityId) {
+      return Promise.reject(new Error("A facility id is required to update a facility's info."));
+    }
+
+    if (!update) {
+      return Promise.reject(new Error("An update is required to update a facility's info."));
+    }
+
+    if (!isPlainObject(update)) {
```

Comment:
Is an empty `update` of `{}` also valid?

Reply:
It won't error, but it still will make a request. So, yes.